server/src/main/java/org/opensearch/index/engine/InternalEngine.java

```diff
@@ -215,6 +212,8 @@ public class InternalEngine extends Engine {
     @Nullable
     private final String historyUUID;
 
+    private final String translogUUID;
```

Comment:
Why `translogUUID` is a part for InternalEngine? We can always fetch it from translogManager, right?